packages/core/test/controls/numericInputTests.tsx

```diff
@@ -519,7 +519,6 @@ describe("<NumericInput>", () => {
         it("shows placeholder text if provided", () => {
             const component = mount(<NumericInput placeholder={"Enter a number..."} />);
 
-            const inputGroup = component.find(InputGroup);
```

Comment:
removed unused var. this was throwing errors in the build, but not failing builds. i guess test code compilation is set up to be more lenient?